screens/Map.js

```diff
@@ -82,35 +91,56 @@ export default class Map extends React.Component {
     });
   };
 
-  handlePress = marker => {
+  setRegionAndSelectLandmark = (event, marker) => {
+    this.setState({
```

Comment:
@a-Daley i think this is what your method was meant to do? i added the `event` param and used `this.setState` instead of `this.regionChange`, but feel free to refactor if you had something else in mind.